code/modules/food_and_drink/plants.dm

```diff
@@ -634,6 +636,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 	heal_amt = 2
 	initial_volume = 100
 	food_effects = list("food_refreshed")
+	brew_result = "capsaicin"
```

Comment:
This ensures 20 units instead of engaging with potency system. Seems like this would make generating smokes a lot easier.